src/components/pages/manage/productions/create/index.js

```diff
@@ -89,6 +101,20 @@ const Create = () => {
     [selectedProductionId],
   );
 
+  const isEditingProduction = useMemo(
+    () =>
+      [
+        createProductionWithEventsMutation.status,
+        mergeProductionsMutation.status,
+        addEventsByIdsMutation.status,
+      ].some((status) => status === QueryStatus.LOADING),
+    [
+      createProductionWithEventsMutation.status,
+      mergeProductionsMutation.status,
+      addEventsByIdsMutation.status,
+    ],
+  );
```

Comment:
We could also use something like this here.

https://react-query.tanstack.com/reference/useIsFetching

Reply:
I don't think we can use this one. Because it only counts the queries being executed, since mutations don't contain a key

Reply:
True!